pennylane_lightning_gpu/_serialize.py

```diff
@@ -36,8 +36,12 @@
     from pennylane_lightning_gpu.lightning_gpu_qubit_ops import (
         LightningGPU_C128,
         LightningGPU_C64,
-        ObsStructGPU_C64,
-        ObsStructGPU_C128,
+        NamedObsGPU_C64,
+        NamedObsGPU_C128,
+        TensorProdObsGPU_C64,
+        TensorProdObsGPU_C128,
+        HamiltonianGPU_C64,
+        HamiltonianGPU_C128,
```

Comment:
Were there many changes needed for the types to get them working here? Could we easily offload these to the Lightning types in a follow-up PR?

Reply:
Yes, these are needed to get them work.